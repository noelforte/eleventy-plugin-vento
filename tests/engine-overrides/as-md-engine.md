---
title: This is a cool page
---

2 + 2 = {{ 2 + 2 }}

{{ title }}
