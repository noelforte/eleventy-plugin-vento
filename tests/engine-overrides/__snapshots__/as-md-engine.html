<p>2 + 2 = 4</p>
<p>This is a cool page</p>
